src/content/pages/programme/proposals.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Writing proposals
+subtitle: A set of tips to help you put together a strong, compelling EuroPython proposal.
+---
+
```

Comment:
Could we add a h1 heading so the page looks like it has a title, e.g. # How to write a proposal